### PR TITLE
Add GRS

### PR DESCRIPTION
**Pull request by gruve-p**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1058,7 +1058,7 @@ To enable lightning, just run:
 export COIN_LIGHTNING=true
 ```
 
-For each `COIN` (i.e. `BTC`, `LTC`) where you want to enable lightning support.
+For each `COIN` (i.e. `BTC`, `LTC`, `GRS`) where you want to enable lightning support.
```
Comment (MrNaif2018) [inline CHANGELOG.md]:
```suggestion
For each `COIN` (i.e. `BTC`, `LTC`) where you want to enable lightning support.
```
Changelogs are historical documents

File: api/ext/moneyformat/currencies.json [hunk 2]
```diff
@@ -325,5 +325,10 @@
         "name": "Tron",
         "divisibility": 6,
         "crypto": true
+    },
+    "GRS": {
+        "name": "GRS",
```
Comment (MrNaif2018) [inline api/ext/moneyformat/currencies.json]:
```suggestion
        "name": "Groestlcoin",
```
It should be full name here ideally (I will fix others later)

File: conf/.env.dev.sample [hunk 3]
```diff
@@ -3,7 +3,7 @@
 # It enables some minimal required functionality
 
 
-BITCART_CRYPTOS=btc,ltc
+BITCART_CRYPTOS=btc,ltc,grs
```
Comment (MrNaif2018) [inline conf/.env.dev.sample]:
```suggestion
BITCART_CRYPTOS=btc,ltc
```

ltc is used in tests usually

File: conf/.env.dev.sample [hunk 4]
```diff
@@ -14,3 +14,5 @@ BTC_LIGHTNING=true
 BTC_NETWORK=testnet
 LTC_LIGHTNING=true
 LTC_NETWORK=testnet
+GRS_LIGHTNING=true
+GRS_NETWORK=testnet
```
Comment (MrNaif2018) [inline conf/.env.dev.sample]:
```suggestion
```

File: conf/.env.sample [hunk 5]
```diff
@@ -10,7 +10,7 @@
 ### Important settings ###
 
 # comma-separated list of currencies to enable, defaults to btc
-BITCART_CRYPTOS=btc,ltc
+BITCART_CRYPTOS=btc,ltc,grs
```
Comment (MrNaif2018) [inline conf/.env.sample]:
```suggestion
BITCART_CRYPTOS=btc,ltc
```

File: conf/.env.sample [hunk 6]
```diff
@@ -20,7 +20,7 @@ LOG_DIR=logs
 
 ### Daemons connection settings ###
 
-# To set them, for each coin, use it's upper name to configure daemon settings, for example, for bitcoin use BTC, for litecoin use LTC, etc.
+# To set them, for each coin, use it's upper name to configure daemon settings, for example, for bitcoin use BTC, for litecoin use LTC, for groestlcoin use GRS, etc.
```
Comment (MrNaif2018) [inline conf/.env.sample]:
```suggestion
# To set them, for each coin, use it's upper name to configure daemon settings, for example, for bitcoin use BTC, for litecoin use LTC, etc.
```

File: conf/terminals.json [hunk 7]
```diff
@@ -11,6 +11,11 @@
             "description": "LTC Daemon",
             "command": "python3 daemons/ltc.py"
         },
+        {
+            "name": "GRS Daemon",
+            "description": "GRS Daemon",
+            "command": "python3 daemons/grs.py"
+        },
```
Comment (MrNaif2018) [inline conf/terminals.json]:
```suggestion
```

File: requirements/daemons/grs.txt [hunk 8]
```diff
@@ -0,0 +1,3 @@
+https://github.com/Groestlcoin/electrum-grs/archive/refs/tags/v4.3.1.zip#egg=electrum-grs[crypto]
```
Comment (MrNaif2018) [inline requirements/daemons/grs.txt]: Please change to the right hash like we discussed

**Review by MrNaif2018 — CHANGES_REQUESTED** [review]
Thank you for your PR! Please check the review comments

File: requirements/daemons/grs.txt (hunk 8 above)
Comment (gruve-p) [reply requirements/daemons/grs.txt]: Done

**Pull request closed without merging**